pandas/tests/indexes/datetimes/test_misc.py

```diff
@@ -408,3 +408,20 @@ def test_isocalendar_returns_correct_values_close_to_new_year_with_tz():
         dtype="UInt32",
     )
     tm.assert_frame_equal(result, expected_data_frame)
+
+
+def test_datetimelike_string():
```

Comment:
ideally could paramaterize this with things that don't match here (e.g. more than just 1 thing).

Also this is not in the right location, search for were we test .equals

Reply:
I will move this in pandas/tests/indexes/datetimes/test_ops.py 

By trying to parametrize I have now more question about the expected behaviors, see comment that follows.